packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/classes_table_diff.dart

```diff
@@ -117,6 +117,14 @@ class _InstanceColumn extends ColumnData<DiffClassStats> {
     }
   }
 
+  @override
+  String getDisplayValue(DiffClassStats classStats) {
+    // Add leading sign for delta values.
+    final value = getValue(classStats);
+    if (dataPart != _DataPart.delta || value <= 0) return value.toString();
+    return '+$value';
```

Comment:
seems like we should also be adding the negative sign when the value is negative

Reply:
when value is negative, the sign is already here